pallets/core_mods/src/modules/offchain_signatures/actions.rs

```diff
@@ -1,51 +1,51 @@
 use super::*;
 
-#[derive(Encode, Decode, scale_info_derive::TypeInfo, Clone, PartialEq, Debug)]
+#[derive(Encode, Decode, scale_info_derive::TypeInfo, Clone, PartialEq, Eq, Debug)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[scale_info(skip_type_params(T))]
 #[scale_info(omit_prefix)]
```

Comment:
What is `omit_prefix` for?

Reply:
To omit the default prefix created by the `scale_info` (file path).